Sources/Models/DetectorType.swift

```diff
@@ -38,7 +38,7 @@ public enum DetectorType {
     //case hashtag
     //case custom
 
-    var textCheckingType: NSTextCheckingResult.CheckingType {
+    public var textCheckingType: NSTextCheckingResult.CheckingType {
```

Comment:
Internal would be good for this too. I may have some future plans that could affect this